libethereum/GasPricer.h

```diff
@@ -40,7 +40,7 @@ enum class TransactionPriority
 	Highest = 8
 };
 
-static const u256 DefaultGasPrice = 20 * shannon;
+static const u256 DefaultGasPrice = 0;
```

Comment:
I'm not sure it make sense to have it now. It's only used in 2 places.

Reply:
so should we set the default value to 0 ?

Reply:
Nevermind. It's used in more places.